commands/errors.go

```diff
@@ -0,0 +1,55 @@
+package commands
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+// FindEntitySummary is thrown when at least, one error is found
+// when finding entities.
+type FindEntitySummary struct {
```

Comment:
Do we need this struct? I can't see if it is used anywhere?

Reply:
Sorry! This was part of the first approach. I'll delete it